checker/src/main/java/org/checkerframework/checker/i18n/qual/Localized.java

```diff
@@ -16,21 +16,20 @@
  * @checker_framework.manual #i18n-checker Internationalization Checker
  */
 @SubtypeOf(UnknownLocalized.class)
-@ImplicitFor(
-        literals = {
-            /* All integer literals */
-            LiteralKind.INT,
-            LiteralKind.LONG,
-            LiteralKind.FLOAT,
-            LiteralKind.DOUBLE,
-            LiteralKind.BOOLEAN,
+@QualifierForLiterals({
+    /* All integer literals */
+    LiteralKind.INT,
+    LiteralKind.LONG,
+    LiteralKind.FLOAT,
+    LiteralKind.DOUBLE,
+    LiteralKind.BOOLEAN,
 
-            /* null should be the bottom type */
-            LiteralKind.NULL
+    /* null should be the bottom type */
+    LiteralKind.NULL
 
-            // CHAR_LITERAL,
-            // STRING_LITERAL,
-        })
+    // CHAR_LITERAL,
```

Comment:
Should we just remove these comments? It's not clear to my why they are here. If you understand, can you add a comment? Later realized what this means and added comment to `All integer literals` above.